src/celestia/celx_internal.h

```diff
@@ -94,7 +95,160 @@ class CelxLua
     CelxLua(lua_State* l);
     ~CelxLua() = default;
 
+    static int localIndex(int n) { return lua_upvalueindex(n); }
+
+    /**** push-on-stack methods ****/
+
+    int push()
```

Comment:
It looks like these method should return `void`.

Reply:
I did it for shorter code. Every push(...) function returns number of pushed elements (I know, for now it is 1 in every case), so I can write
`return push(...);`
instead of
`push();
return 1;`

Reply:
so more proper to write as return push(...)

Reply:
I dont understand. This is as I write.

Reply:
Ahhh, you want to return values of **your** push-es. okay.